build.py

```diff
@@ -1857,6 +1857,12 @@ def core_build(
         os.path.join(install_dir, "python"),
     )
 
+    cmake_script.mkdir(os.path.join(install_dir, "python"))
+    cmake_script.cp(
+        os.path.join(repo_install_dir, "python", "tritonfrontend*.whl"),
```

Comment:
Is this needed? `triton*.whl` right above this

Reply:
Thank you for the catch. Just removed it.